session/kx_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2016,2017 Company 0, LLC.
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package session
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/companyzero/zkc/zkidentity"
+)
+
+var (
+	alice   *zkidentity.FullIdentity
+	bob     *zkidentity.FullIdentity
+	aliceKX *KX
+	bobKX   *KX
+)
+
+var mtx sync.Mutex
```

Comment:
Boo!

Reply:
Hi Marco,

This lock is only used during the test to synchronise access to log(). What do you suggest instead?